### ISSUE TITLE
I've made a Brew, saved it. When I clicked in my user name, everything was lost. 

### ISSUE THREAD

**Issue by Luiztomikawa** [naturalcrit/homebrewery#725]
Today I spent my entire day making a Brew... then when I was ready to go to sleep I clicked in my name and boom! My entire thing was lost... I clicked in the "Recent Brews" but any of the options weren't available to be clicked on! Then I made again more one page and "Saved it" and yet again when I go away from the page, everything disappear without a trace... I saw someone saying about login again because the user somethimes is screwed... but I got back to the "Natural crit" site... and there is not even a single button to log in or anything... so I really don't know what to do. I want my Sunday back! T-T

**Comment by PSDNDFUN** [comment]
Hmm I could be wrong but I remember saving it first so that it gave you a special address then typing in the stuff and letting it auto save.  Unless they made it so that it doesn't even really save anymore but I just worked on one so I don't think that is the case.

**Comment by Luiztomikawa** [comment]
So, I just don't know how the saving on this site works, but luckily I've sended the link to the brew to some friends, so after much despair they reminded me that I had sended they the link! So now I can open it because I have the link saved in a Notepad file. But that is kinda ridiculous how the site doesn't provide a form for you to access your own project, I was only able to do so because I saved the link externally!
